README.md

```diff
@@ -5,36 +5,21 @@
 
 It allows developers to quickly and in a simple manner define and create large number of objects whose attributes have randomly selected values from the configured set.
 
-It can be used for following:
+It can be used for:
```

Comment:
Consider stating that its two main purposes are to be used in integration tests and as a data source for Berserker.